plugins/ipam/host-local/backend/disk/lock.go

```diff
@@ -1,3 +1,4 @@
+// +build !windows
```

Comment:
>+build !windows [](start = 3, length = 15)

what made you include this instead of renaming it to lock_linux.go like you did with the other files?

Reply:
because this way the `host-local` tests can run on my Mac :-)